.shopify-build/polaris-system-integration-test.yml

```diff
@@ -0,0 +1,29 @@
+containers:
+  default:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.18.0-yarn-1.21.1
+  styleguide:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.13.0-yarn-1.21.1
+
+shared:
+  build-polaris: &build-polaris
+    label: 'Build polaris'
+    container: default
```

Comment:
https://shopify-build.docs.shopify.io/jobs/sandboxes/ says that if you don't specify a container it'll used the one called 'default' so perhaps remove this as we only ever have one?